docs/developer/decision-records/2022-04-21-dpf-blob-data-transfer/README.md

```diff
@@ -0,0 +1,38 @@
+# Blob storage transfer
+
+ADR describing the blob storage transfer end to end flow between 2 participants.
+
+## Description
+
+The data-plane-azure-storage extension can be used on DPF to support blob transfers.
+A client can trigger a blob transfer on the consumer side via the Data Management API.
+The client needs to use the managedResources=true option in its HTTP request, otherwise the transfer will not work. Client would not be aware that an error occurred, it would just never see the transferProcess with completed state when polling for the result.   
```

Comment:
These are really two different issues.
- if using `managedResources=false`, the SAS URL is not generated, so the transfer will not work.
- the provider does not send to the consumer any message after the transfer was performed (succeeded or failed). The consumer only polls for the result. Therefore, if the transfer fails for *any* reason, the consumer is not aware of it and keeps polling indefinitely.

We should link to EDC issues for those/create them if necessary

Reply:
Created
- https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/1242
- https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/1241